cmd/lakectl/cmd/root.go

```diff
@@ -140,6 +147,10 @@ const (
 	allowEmptyMsgFlagName = "allow-empty-message"
 	fmtErrEmptyMsg        = `commit with no message without specifying the "--allow-empty-message" flag`
 	metaFlagName          = "meta"
+
+	defaultMaxRetries       = 3
+	defaultMaxRetryInterval = 1 * time.Second
+	defaultMinRetryInterval = 200 * time.Millisecond
```

Comment:
I don't understand these numbers.  Suppose the HTTP server never sends a "retry after" header (I don't think lakeFS does).  Then we have 3 retries, starting with 0.2 s and going up to 1 second.  But 3 retries will sleep for 0.2s and then for 0.4s.  When does this max retry interval ever come into play?

If we had *6* retries, then IIUC our wait sequence would be (in seconds) 0.2, 0.4, 0.8, 1.0, 1.0, 1.0.

For example, AFAICT [this](https://github.com/hashicorp/go-retryablehttp/blob/4165cf8897205a879a06b20d1ed0a2a76fbb6a17/client.go#L51-L53) is the default configuration.

Reply:
>If we had 6 retries, then IIUC our wait sequence would be (in seconds) 0.2, 0.4, 0.8, 1.0, 1.0, 1.0

Yes, that's accurate. I'm not sure why this is a problem, but I don't mind using the package defaults instead.

Reply:
Done